app/store/kg.ts

```diff
@@ -53,7 +53,7 @@ export const useKGStore = createPersistStore(
         }
         const server = kgInfo.servers[0];
         this.updateConfig((config: KGConfig) => {
-          config.connectionArgs.host = server.server;
+          config.connectionArgs.host = server.address;
```

Comment:
Here we should use server.address as server.server is server name that is for display